pkgs/desktops/gnome/extensions/pano/default.nix

```diff
@@ -0,0 +1,96 @@
+{ lib
+, stdenv
+, mkYarnModules
+, fetchFromGitHub
+, atk
+, cogl
+, glib
+, gnome
+, gtk3
+, libgda
+, pango
+, nodePackages
+}:
+
+let
+  version = "v3";
+  uuid = "pano@elhan.io";
+in
+stdenv.mkDerivation rec {
+  pname = "gnome-shell-extension-pano";
+  inherit version;
+
+  src = fetchFromGitHub {
+    owner = "oae";
+    repo = "gnome-shell-pano";
+    rev = version;
+    hash = "sha256-LVEtxh/+Zjx0TXcP2+7SeW7jKDw9baoea35MQpNfnmQ=";
+  };
+
+  nativeBuildInputs = [
+    nodePackages.rollup
+  ];
+
+  buildInputs = [
+    atk
+    cogl
+    glib
+    gnome.gnome-shell
+    gnome.mutter
+    gtk3
+    libgda
+    pango
+  ];
+
+  nodeModules = mkYarnModules {
+    inherit pname version;
+    name = "${pname}-modules-${version}";
+    packageJSON = ./package.json;
```

Comment:
Is this necessary? I would expect the lockfile to be sufficient.

Reply:
Yes, it is necessary, all of those are required arguments. At least I rewrote it to build `yarn.nix` on the fly. I would appreciate your feedback.

Reply:
I meant something like this: https://github.com/NixOS/nixpkgs/blob/0b00606cf7509ab568e541ac00ce6a4e605ee95b/pkgs/tools/misc/sharedown/default.nix#L88-L90

Reply:
Thanks! It should be fixed now.